app/internal/emotion.py

```diff
@@ -0,0 +1,64 @@
+import text2emotion as te
+from typing import List, Union
+
+from app.config import (
+    CONTENT_WEIGHTS,
+    LEVEL_OF_SIGNIFICANCE,
+    TITLE_WEIGHTS)
+
+
+EMOTIONS = [("Happy", "&#128515"),
+            ("Sad", "&#128577"),
+            ("Angry", "&#128544"),
+            ("Fear", "&#128561"),
+            ("Surprise", "&#128558")]
+
+
+def dominant_emotion(title: str, content: str) -> Union[List]:
+    """
+    get text from event title and content and return
+    the dominant emotion, emotion score and smiley code
+    """
+    dominant_emotion = [None, 0, None]
+    duplicate_dominant_flag = 0
+    title_emotion = te.get_emotion(title)
+    if content is not None:
+        content_emotion = te.get_emotion(content)
+    for emotion, code in EMOTIONS:
```

Comment:
Maybe we should export the content of this for to another function? :)

Reply:
did it :)
split to `get_weights `and `score_comp`